### PR TITLE
GOOGLEDOCS-435 : Update GDOCS build plan's "Publish" stage to *also* …

### PR DESCRIPTION
**Pull request by DenisGabriela**
…copy to S3 bucket

   - updated project with scm details
   - added release-plugin to project

### CODE REVIEW

File: googledrive-repository/pom.xml [hunk 1]
```diff
@@ -215,8 +214,7 @@
                                 </goals>
                                 <configuration>
                                     <tasks>
-                                        <copy file="src/main/customResponseContextFiles/slingshotDocLibCustomResponse/google-customResponse-context.xml"
-                                              tofile="src/main/amp/config/alfresco/subsystems/googledocs/drive/google-customResponse-context.xml"/>
+                                        <copy file="src/main/customResponseContextFiles/slingshotDocLibCustomResponse/google-customResponse-context.xml" tofile="src/main/amp/config/alfresco/subsystems/googledocs/drive/google-customResponse-context.xml" />
```
Comment (montgolfiere) [inline googledrive-repository/pom.xml]: Is this just formatting change ?

File: googledrive-repository/pom.xml [hunk 2]
```diff
@@ -327,8 +324,7 @@
                                 </goals>
                                 <configuration>
                                     <tasks>
-                                        <copy file="src/main/customResponseContextFiles/registerSlingshotCustomResponse/google-customResponse-context.xml"
-                                              tofile="src/main/amp/config/alfresco/subsystems/googledocs/drive/google-customResponse-context.xml"/>
+                                        <copy file="src/main/customResponseContextFiles/registerSlingshotCustomResponse/google-customResponse-context.xml" tofile="src/main/amp/config/alfresco/subsystems/googledocs/drive/google-customResponse-context.xml" />
```
Comment (montgolfiere) [inline googledrive-repository/pom.xml]: Why put onto one line (is this this just an auto-formatting change) ?

Same for other occurrences ...

Comment (DenisGabriela) [reply googledrive-repository/pom.xml]: Yes, is auto-formatting change :(

Comment (DenisGabriela) [reply googledrive-repository/pom.xml]: Thanks for the quick review  💃

**Pull request merged**